### ISSUE TITLE
Introduce prefix search in Firefox sidebar

### ISSUE THREAD

**Issue by darpas70** [seeba8/yet-another-pinboard-extension#50]
Dear Sebastian,
it would be in my opinion very useful if you added the prefix search (based on shortcuts: r,t,u,n) in the Firefox sidebar search. At present this only works in the address bar. Thank you for evaluating this proposal.
Best regards

**Comment by darpas70** [comment]
Hello Sebastian,
I am not sure that I opened correctly this pull request (I am not familiar with the Github process). Please let me know if I need to insert it differently. Thank you in advance
Best regards

**Comment by seeba8** [comment]
Ciao @darpas70 
I think I may have implemented it. If you want, please test it. It is not yet pushed to the addon stores since I want to test it myself a bit as well. You can test it the following way:
Create a new profile for testing or uninstall the addon first.
Then, go to `about:debugging#/runtime/this-firefox`. Then click on Load Temporary Addon. Unzip the attachment here and load the addon as a temporary addon by selecting the file "manifest.json" in the root folder.
[dist.zip](https://github.com/seeba8/yet-another-pinboard-extension/files/4808005/dist.zip)

I've added the prefixed search in the popup as well.

It may have also solved your search problem for ristorante, by the way. I think the cause here was that the search was case sensitive. Would be cool if you could try that again. In chrome, you test by going to `chrome://extensions/`, turning on  the developer mode using the toggle in the top right, and then click "Load unpacked". Here, you need to select the whole folder `dist` (the one that contains the manifest.json file).

**Comment by darpas70** [comment]
Hello Sebastian,
thank you for implementing my proposal. I have done some tests and I report the following:
1) In Firefox everything seems to work properly. The search based on prefix now works in sidebar, address bar and popup for each string that I tried.
2) In Chrome, oddly the issue with the tag "ristoranti" is still present in the address bar, whereas it works properly in the popup.
By the way, note that I did not perform comprehensive and systematic tests but rather just normally used for some time the new extension during my browsing.
Thank you very much for your help
Best regards
